install/README.md

```diff
@@ -7,3 +7,17 @@ To use it, just run the installation script in your terminal **as root**, like s
 ```bash
 sudo su -c 'curl -fsSL https://raw.githubusercontent.com/bluerobotics/companion-docker/master/install/install.sh | bash'
 ```
+
+# Using different versions or custom builds
+To use a different remote or version, you can se the following environment variables:
+- `REMOTE`: Where the files are, E.g: https://raw.githubusercontent.com/patrickelectric/companion-docker
+- `VERSION`: Branch (If using GitHub) or folder (If using HTTP server) to be used.
+
+Remember that to do that, you need to set the environment variables as root:
+```sh
+sudo su
+export VERSION=install-fix
+export REMOTE=https://raw.githubusercontent.com/patrickelectric/companion-docker
+# You can also change the install URL to use a different source for the install script
+curl -fsSL https://raw.githubusercontent.com/patrickelectric/companion-docker/test/install/install.sh | bash
```

Comment:
```suggestion
curl -fsSL https://raw.githubusercontent.com/patrickelectric/companion-docker/install-fix/install/install.sh | bash
```

Reply:
maybe, idk if the comment above is the point here

Reply:
I'll move it to use a clear name